client/extensions/wp-super-cache/main.jsx

```diff
@@ -24,6 +25,7 @@ const WPSuperCache = ( { site, tab } ) => {
 			case Tabs.CONTENTS:
 				return <ContentsTab isMultisite={ site.is_multisite } />;
 			case Tabs.PRELOAD:
+				return <PreloadTab site={ site } />;
```

Comment:
`site` doesn't need to be passed as a prop here since it doesn't appear that it is being used by the component.